broadinstitute/consent
# [DUOS-228] Handle invalid dar ids better

## Addresses
https://broadinstitute.atlassian.net/browse/DUOS-228

## Changes
* Handle the lookup of the dar by document id better. If we can't find it, or if it is not an id we can look up, return a 404. Otherwise, return the exception through the normal exception response mechanism. 
* Added tests.
* Lots of minor formatting cleanup for the files touched.
* While I'm at it ... addressing some long-standing test logging problems in mongo and mockserver.

## Manual Testing
* Spin up and go to the dar lookup page which is `Data_Access_Request/get_api_dar_id`
* Try entering gibberish into the id field and ensure that you get a 404. 
* Enter a real id (look one up from duos dev) and ensure that you still get a valid document.
